lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -953,277 +974,236 @@
 
     // Intl.NumberFormat, Number.prototype.toLocaleString
     var NumberFormat = (function () {
+        if (InitType !== "Intl" && InitType !== "Number") {
+            return;
+        }
         // Keep these "enums" in sync with lib/Runtime/PlatformAgnostic/Intl.h
-        const NumberFormatStyle = setPrototype({
+        const NumberFormatStyle = {
             DEFAULT: 0, // "decimal" is the default
             DECIMAL: 0, // Intl.NumberFormat(locale, { style: "decimal" }); // aka in our code as "number"
             PERCENT: 1, // Intl.NumberFormat(locale, { style: "percent" });
             CURRENCY: 2, // Intl.NumberFormat(locale, { style: "currency", ... });
-        }, null);
-        const NumberFormatCurrencyDisplay = setPrototype({
```

Comment:
>setPrototype({ [](start = 44, length = 14)

Doing away with the idea that an enum should be a null-proto object? Lookup on values not present would be slower. Otherwise that should be fine.

Reply:
I might add this back once I determine if this can be tainted by setting Object.prototype to a proxy.